### DISCUSSION TITLE
Commit not reflected on grass(?)

### DISCUSSION THREAD

**Discussion by Jasonlee1995** [community/community#38641] [Other Feature Feedback, Questions, & Ideas]
Hi, I think I observe some weird thing, which is the commit I did is not reflected on grass(?).

Just to make things simple, I append images for more details.

<img width="700" alt="image" src="https://user-images.githubusercontent.com/49643709/200747338-0a0219a9-4d56-446c-9da2-793695ffd200.png">

Today, I did 1 commit.

<img width="700" alt="image" src="https://user-images.githubusercontent.com/49643709/200745561-a01f44fd-5912-4fa8-8c5b-166e6c9c4101.png">

I notice that my commit is not reflected on my grass(?).

<img width="700" alt="image" src="https://user-images.githubusercontent.com/49643709/200746516-4b77f174-b6ed-4141-976f-f3303c8465b4.png">
<img width="700" alt="image" src="https://user-images.githubusercontent.com/49643709/200746548-eb83a32b-3121-4bef-8ef8-52f1f0d7afbd.png">

In contribution activity, it summarizes that I have only 2 commits on the repo I commit, but there are 3 commits in details.

I hope why it happened cause I never experience this kind of things before.

Any solutions for this phenomena?

**Comment by radar-blip** [comment]
Sometimes, contributions dont show up right away, maybe because of time-zone or something.
Anyway, I am seeing  you made 1 commit today. 

![commit](https://user-images.githubusercontent.com/39196767/200748945-edeaf741-bc25-42ae-a656-9ecbf1d542be.png)

**Reply by Jasonlee1995** [reply]
Thanks for the answer and I found some more weird things :(

<img width="1439" alt="image" src="https://user-images.githubusercontent.com/49643709/200750325-33927c0c-1281-47f2-8d9b-bf6329f198f7.png">

If I get into chrome private mode and select my username and search, I found that commit is properly reflected the commit.
(right of the figure)

But interesting thing is that the current tab I'm using it does not reflect the commit I did.
(left of the figure)

What's the difference between these two?? why things visualize differently??

Of course I refreshed my pages to check commit is properly reflected.

**Reply by radar-blip** [reply]
I'm not exactly sure why it shows different, but your commit is there. The current tab will catch up soon. 😄
